src/tools/chip-cert/README.md

```
@@ -169,6 +169,29 @@ attestation certificate chain that was just created:
 openssl verify -CAfile Chip-PAA-Cert.pem -untrusted Chip-PAI-Cert.pem Chip-DAC-Cert.pem
 ```
 
+## PDC Identity Usage Examples
+
+Example commands to generate a PDC identity (a constrained self-signed
+certificate):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)